x-pack/plugin/ent-search/qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/xpack/application/FullClusterRestartIT.java

```diff
@@ -53,11 +49,11 @@ protected ElasticsearchCluster getUpgradeCluster() {
         return cluster;
     }
 
+    @Ignore("https://github.com/elastic/elasticsearch/issues/104470")
```

Comment:
We should be using the `@AwaitsFix` annotation here instead.

Reply:
Ok, I updated it.